src/objects/zcl_abapgit_object_intf.clas.abap

```diff
@@ -520,14 +520,28 @@ CLASS zcl_abapgit_object_intf IMPLEMENTATION.
 
 
   METHOD zif_abapgit_object~deserialize.
-    DATA: lt_source TYPE rswsourcet,
-          ls_clskey TYPE seoclskey,
-          ls_intf   TYPE ty_intf.
+    DATA:
+      lt_source          TYPE rswsourcet,
+      ls_clskey          TYPE seoclskey,
+      ls_intf            TYPE ty_intf,
+      lt_description     TYPE zif_abapgit_oo_object_fnc=>ty_seocompotx_tt,
+      lt_description_sub TYPE zif_abapgit_oo_object_fnc=>ty_seosubcotx_tt.
 
     IF iv_step = zif_abapgit_object=>gc_step_id-abap.
       " HERE: switch with feature flag between XML and JSON file format
       IF mv_aff_enabled = abap_true.
         ls_intf = read_json( ).
+
+        lcl_aff_metadata_handler=>deserialize_translation(
+          EXPORTING
+            it_files           = mo_files
+          IMPORTING
+            et_description     = lt_description
+            et_description_sub = lt_description_sub ).
+
+        APPEND LINES OF lt_description TO ls_intf-description.
+        APPEND LINES OF lt_description_sub TO ls_intf-description_sub.
```

Comment:
@mbtools I just wonder where to put the translated object description? Didn't found the place in the abapGit data type, here `ls_intf`.
<img width="684" alt="Screenshot 2024-03-08 at 10 09 40" src="https://github.com/abapGit/abapGit/assets/38354196/2a17f995-1342-49a0-8da8-211d925ba4f3">

Reply:
splitted the question into issue #6846, as I assume there is not an super easy answer to this one. So we can move with this pr.

Reply:
Should we add the information this PR does not handle interfaces' object description to the PR's description?